src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -1,29 +1,76 @@
 package traffic_simulation
 
+// Imports for parsing library
+import com.univocity.parsers.common.record.Record
+import com.univocity.parsers.csv.CsvParser
+import com.univocity.parsers.csv.CsvParserSettings
+import java.io.*
+
 fun main(args: Array<String>) {
 
-    scenario(testList, testRoad)
+    val testRoad : RoadNetwork = RoadNetwork(capacity = 8.0)
+
+    val driveInterest : MutableList<Vehicle> = parseInputOfCSV(fileName = "driveInterest.csv")
+
+
+
+   // scenario(testList, testRoad)
 }
 
 fun scenario(vehicleList: List<Vehicle>, road: RoadNetwork){
     road.calculateTraffic(vehicleList)
 }
 
 
-val testRoad: RoadNetwork = RoadNetwork(8.0)
 
-// Creation of sufficient cars for local testing without using CSV input for now
-val BMW     : Vehicle = Vehicle(1.0)
-val AUDI    : Vehicle = Vehicle(1.0)
-val VOLVO   : Vehicle = Vehicle(1.0)
-val VW      : Vehicle = Vehicle(1.0)
-val TESLA   : Vehicle = Vehicle(1.0)
-val SUZUKI  : Vehicle = Vehicle(1.0)
-val HONDA   : Vehicle = Vehicle(1.0)
-val FERRARI : Vehicle = Vehicle(1.0)
-val PORSCHE : Vehicle = Vehicle(1.0)
-val DAIHATSU: Vehicle = Vehicle(1.0)
-val FORD    : Vehicle = Vehicle(1.0)
-val OPEL    : Vehicle = Vehicle(1.0)
```

Comment:
Nicht vergessen, diese Liste beizubehalten. Muss nicht hier sein (im Gegenteil, in `scenario` gefiele sie mir besser).

[Oben](https://github.com/RF-Buerky/BBFW-Project/pull/41#discussion_r131409043) hast du geschrieben:

> ok. könnte man später ja auch in einen Test verschieben, oder?

Das kann auch ein Test werden, ja. Aber unabhängig davon, will ich, dass es für die Entwickler einfach ist, einen vollen Programmlauf aus der `main` Methode heraus laufen zu lassen, _ohne_ dass dafür Parsing einwandfrei funktionieren muss.